jenkins/toggle-webhook-processor.py

```diff
@@ -0,0 +1,63 @@
+import boto3
+import sys
+import logging
+
+
+def main():
+    try:
+        # connect to aws lambda
+        client = boto3.client('lambda')
+    except Exception, e:
+        # if connection failed, log error
+        logging.error(e)
+        sys.exit(1)
+
+    # ensure the number of arguments is correct
+    num_args = len(sys.argv)
```

Comment:
Rather than writing your own argument parsing code, you should use a common python module. Couple good choices: 
* Looks like the other scripts in this directory use "argparse"
* The pattern that we are generally following now, although I think you will need to add this requirement, is to use "[click](http://click.pocoo.org/5/)". Here's [an example](https://github.com/edx/tubular/blob/master/tubular/scripts/jenkins_trigger_build.py). FWIW This is the choice that I personally prefer.